chore(@edit_network): verification  for checkbox text and successful connection messages

anastasiyaig opened the pull request:
- Added verification for text string for checkbox
- Added verifications for error messages in input fields
- Added verifications for successful messages when connection established
- Fixed the problem with clicking button too fast

<img width="1840" alt="Screenshot 2023-10-05 at 16 09 53" src="https://github.com/status-im/desktop-qa-automation/assets/82375995/9bd5b61d-f381-44ec-9a31-937cd434801d">

Valentina1133 commented on gui/screens/settings.py:
```diff
@@ -479,6 +481,21 @@ def check_acknowledgement_checkbox(self, value: bool):
         self._network_acknowledgment_checkbox.set(value)
         return self
 
+    @allure.step('Verify the text for consent when changing RPC urls')
```
Maybe it's better to name this step and also two next steps 'Get the text...'? Because here we are not verifying/checking yet, just getting text (verifying in test itself)

anastasiyaig replied:
sure, will fix.

Valentina1133 reviewed (commented):
I checked test, it fails with error:
>           assert edit_network_form.get_edit_network_main_json_rpc_url_value() == WalletNetworkDefaultValues.ETHEREUM_LIVE_MAIN.value
E           AssertionError: assert 'https://eth-archival.gateway.pokt.network/v1/lb/************************' == 'https://eth-archival.gateway.pokt.network/v1/lb/********'
E             - https://eth-archival.gateway.pokt.network/v1/lb/********
E             + https://eth-archival.gateway.pokt.network/v1/lb/************************
E             ?                                                         ++++++++++++++++

anastasiyaig commented:
> I checked test, for some reason it fails with error:
> 
> > ```
> >       assert edit_network_form.get_edit_network_main_json_rpc_url_value() == WalletNetworkDefaultValues.ETHEREUM_LIVE_MAIN.value
> > ```
> 
> E AssertionError: assert 'https://eth-archival.gateway.pokt.network/v1/lb/**' == 'https://eth-archival.gateway.pokt.network/v1/lb/' E - https://eth-archival.gateway.pokt.network/v1/lb/ E + https://eth-archival.gateway.pokt.network/v1/lb/****************** E ? ++++++++++++++++

i have no idea what is happening, perhaps it is issue in app. I will investigate

anastasiyaig commented:
@Valentina1133 the problems are fixed now in this PR:
- assert error was correct, right now we have more star symbols (perhaps we changed the key or something) 
- the problem with not clicking button is also fixed
- fixed steps names as u suggested
- added verifications for successful messages too

Please take another look

StateOf-Vlado reviewed (approved):
Perfect!
